pkgs/by-name/st/stackql/package.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, testers
+, stackql
+}:
+
+buildGoModule rec {
+  pname = "stackql";
+  version = "0.5.643";
+
+  src = fetchFromGitHub {
+    owner = "stackql";
+    repo = "stackql";
+    rev = "v${version}";
+    hash = "sha256-9W6bEI+5Q0Kgbd14sWKde3I6WIVz1ZxsXmR009mOPog=";
+  };
+
+  vendorHash = "sha256-xcly4jtdUkx/s+YWYFBVeuI2kQBu2oqbLN9ZKkHppkA=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/stackql/stackql/internal/stackql/cmd.BuildMajorVersion=0"
+    "-X github.com/stackql/stackql/internal/stackql/cmd.BuildMinorVersion=5"
+    "-X github.com/stackql/stackql/internal/stackql/cmd.BuildPatchVersion=643"
```

Comment:
This seems hard to keep in sync, especially during auto updates with bots like `r-ryantm`. Consider using `lib.splitVersion` to generate these based on the `version` so that it remains in sync.